lib/ansible/modules/apt.py

```diff
@@ -38,6 +38,7 @@
       - Default is not to update the cache.
     aliases: [ update-cache ]
     type: bool
+    default: false
```

Comment:
2 things:
- The `default` entry is not required when it is ... the default (`None`).
- `false` is not `None`, which is the actual current default, that is why you get a mismatch.

Reply:
FWIW we do rely on `None` (=user hasn't specified the option) being the default in the code where we check for `False` (=user explicitly specifed `"no"`):
https://github.com/ansible/ansible/blob/4a96b3d5b41fe9f0d12d899234b22e676d82e804/lib/ansible/modules/apt.py#L1268-L1271

So we cannot change the default to `False` with the current implementation.

Reply:
Alright, thank you for your time and the information. I didn't know about this requirement.
Cheers,
Felix